routes/indexRoute.js

```diff
@@ -1,9 +1,8 @@
 import express from "express";
+import { index } from "../controllers/indexController.js";
 
 const indexRoute = express.Router();
 
-indexRoute.get("/", (req, res) => {
-    res.render("./listings/welcome.ejs");
-});
+indexRoute.get("/", index);
```

Comment:
you can create one more constant file call Routes.js and have all the routes exported as a constant in there. One file for Routes that is hit from the browser and one is for the EjsTemplateRoutes

Reply:
will do ✅